tck/src/main/java/org/eclipse/microprofile/config/tck/EmptyValue/EmptyValuesTest.java

```diff
@@ -32,15 +33,14 @@
 
 import javax.inject.Inject;
 
-import static org.testng.Assert.assertEquals;
-
 public class EmptyValuesTest extends Arquillian {
 
     private static final String EMPTY_PROPERTY = "my.empty.property";
     private static final String PROP_FILE_EMPTY_PROPERTY = "my.empty.property.in.config.file";
     public static final StringAsset EMPTY_STRING_ASSET = new StringAsset(PROP_FILE_EMPTY_PROPERTY + "=");
 
     @Deployment
+    @ShouldThrowException(DeploymentException.class)
```

Comment:
I believe this won't throw the expected exception because `EmptyValuesBean#stringValue` sets a default in the annotation.

Reply:
The default is an empty value though so this is to verify this empty value won't make the property exist.